.github/workflows/test_models_part2.yaml

```diff
@@ -0,0 +1,156 @@
+name: Test Skyportal Model Logic - pt 2
```

Comment:
The model logic runs in <20 mins, so probably easiest to keep these together for now.

Reply:
It was actually like 40 minutes on the last build I checked.

Reply:
e.g. https://github.com/skyportal/skyportal/runs/3167096342

Reply:
I missed the container initialization; 15 minutes?! That's nuts.

Reply:
Playing around with some options in https://github.com/skyportal/skyportal/pull/2148

Reply:
Down to 15s :man_shrugging:

Reply:
The run above might have been a fluke; looks like that step doesn't usually take that long.